Wrappers/Python/cil/optimisation/operators/GradientOperator.py

```diff
@@ -90,16 +91,16 @@ def __init__(self, domain_geometry, method = 'forward', bnd_cond = 'Neumann', **
         if correlation == CORRELATION_SPACE and domain_geometry.channels > 1:
             #numpy implementation only for now
             backend = NUMPY
-            warnings.warn("Warning: correlation='Space' on multi-channel dataset will use `numpy` backend")
+            logging.warning("Warning: correlation='Space' on multi-channel dataset will use `numpy` backend")
 
         # Complex data will use numpy backend
         if domain_geometry.dtype in [np.complex, np.complex64]:
             backend = NUMPY
-            warnings.warn("Warning: Complex geometries will use `numpy` backend")
+            logging.warning("Warning: Complex geometries will use `numpy` backend")
         
         if method != 'forward':
             backend = NUMPY
-            warnings.warn("Warning: method = {} implemented on `numpy` backend. Other methods are backward/centered.".format(method))            
+            logging.warning("Warning: method = {} implemented on `numpy` backend. Other methods are backward/centered.".format(method))            
```

Comment:
as per discussion, these should be `info`

Reply:
Changed `warnings` to `info`.

The output of 

```bash

if __name__ == "__main__":
    
    from cil.optimisation.operators import GradientOperator
    from cil.framework import ImageGeometry
    import logging

    logger = logging.getLogger()
    logger.setLevel(logging.INFO)  

    ig = ImageGeometry(3,4)
    Grad = GradientOperator(ig)

```
is

```bash
Message: 'Initialised GradientOperator with C backend running with '
Arguments: (20, ' threads')
```